2_adventures/001/kotp/04_automated_testing_example/spec/fizz_buzz_spec.rb

```diff
@@ -0,0 +1,53 @@
+require_relative 'spec_helper'
+require 'fizz_buzz'
+
+describe FizzBuzz do
+  describe 'Single Numbers' do
+    it 'must return 1 for 1' do
```

Comment:
I wonder if I got in the habit of simply ensuring a space is the first character if I would have it as exactly how I want.  Or perhaps I should simply make a custom reporter and lose that.  I don't care about the test part, I know it is a test.  The number of course is helpful and of course the 'must' is simply redundant.